client/components/dialog/style.scss

```diff
@@ -73,7 +73,6 @@
 		color: $gray-dark;
 		font-size: 1.375em;
 		font-weight: 600;
-		height: 2em;
```

Comment:
I removed this height so the title of the dialog can be displayed in 2 lines if it doesn't fit the available width (i.e. in mobile devices)